randovania/games/prime1/json_data/header.txt

```diff
@@ -2,14 +2,34 @@
 Templates
 
 * Shoot Super Missile:
-      Charge Beam and Missile ≥ 5 and Power Beam and Super Missile
+      Charge Beam and Missile ≥ 5 and Power Beam and Super Missile and Can Use Arm Cannon
 
 * Have all Beams:
-      Ice Beam and Plasma Beam and Power Beam and Wave Beam
+      Ice Beam and Plasma Beam and Power Beam and Wave Beam and Can Use Arm Cannon
 
 * Heat-Resisting Suit:
       Gravity Suit or Phazon Suit or Varia Suit
 
+* Can Use Arm Cannon:
+      Combat Visor or Thermal Visor or X-Ray Visor
+
+* Can Shoot Any Beam:
+      All of the following:
+          Can Use Arm Cannon
+          Ice Beam or Plasma Beam or Power Beam or Shoot Wave Beam
```

Comment:
Is there a reason this is "Shoot Wave Beam" and not just Wave Beam?

Reply:
No, it was an accident